app/less/default/core/variables.less

```diff
@@ -123,7 +123,7 @@
 @thumb-color:                   @gray-light;
 
 @thumb-radius:                  8px;
-@thumb-height:                  12px;
+@thumb-height:                  8px;
```

Comment:
Could we not leave the thumb the same size so it's more obvious which part to click and drag like in spotify?

![screen shot 2015-06-03 at 12 31 15](https://cloud.githubusercontent.com/assets/7836746/7959006/e88f7418-09ec-11e5-800e-d7bd6867a319.png)